core/src/test/java/io/aiven/kafka/tieredstorage/fetch/cache/DiskChunkCacheMetricsTest.java

```diff
@@ -89,7 +89,8 @@ void metrics() throws IOException, JMException, StorageBackendException {
         final DiskChunkCache diskChunkCache = new DiskChunkCache(chunkManager, time);
         diskChunkCache.configure(Map.of(
             "size", size1,  // enough to put the first, but not both
-            "path", baseCachePath.toString()
+            "path", baseCachePath.toString(),
+            "retention.ms", String.valueOf(Duration.ofSeconds(10).toMillis())
```

Comment:
I believe the idea of the test was different. It was intended to test that only 1 chunk is deleted because the cache size reached the limit and because of `Window TinyLfu` it is not possible to say first or second chunk will be deleted. Specifying `retention.ms` I believe we are missing the case when the metrics are reported correctly for a single chunk since there is a high chance that both will be deleted.

Reply:
We have seen failing test cases (unfortunately too old to reference) that are not dependent on time, meaning that even if we increase the timeout to more than 30s it will not help: the eviction already failed (for some reason; could be the file not exist exception), and the metrics will not tick.
This test is not meant to check if the deletion is working properly (I think it's out of the scope of this test), but to check that the deletion metric is reported.
So expanding the cache configuration to trigger deletion in a more consistent way seem like a fair trade to remove flakiness (otherwise we wait for the flaky test to trigger and hope to get enough logs to troubleshoot -- this hasn't been the case.. CI logs only show "test failed" without including logs.. maybe something else to fix?)